### PR TITLE
RS-476: Update price page

### PR DESCRIPTION
**Pull request by AnthonyCvn**
- ❗Updated homepage title and subtitle
- Added new price list and waiting list forms
- Created a new cloud solution page

### CODE REVIEW

File: src/pages/solutions/cloud/index.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+export default function CloudSolution(): JSX.Element {
+  return (
+    <Layout
+      title="Cloud Time Series Object Storage"
+      description="High-performance cloud storage for data of any size—images, text, sensor data, and more."
+    >
+      <main>
+        <SimpleHeaderWithForm
+          title="Cloud Time Series Object Storage"
```
Comment (atimin) [inline src/pages/solutions/cloud/index.tsx:18]: All well and good, but we're not using Azure for the SaaS. We build it on Google Cloud. :hugs:

**Review by atimin — CHANGES_REQUESTED** [review]

File: src/components/PricingTable/index.tsx [hunk 2]
```diff
@@ -13,15 +14,27 @@ export default function PricingTable() {
   return (
     <section>
       <div className={clsx("row", styles.pricingTable)}>
+        {/* Community Plan */}
         <PricingPlan
```
[the remark this answers is not in the document]
Comment (AnthonyCvn) [reply src/components/PricingTable/index.tsx:18]: I tried but looks a bit long, sometimes it's good to have text with the question mark icon for more details, I'll think of another solution.

Screenshot 2024-10-02 at 14 16 00

[the remark this answers is not in the document]
Comment (AnthonyCvn) [reply src/components/PricingTable/index.tsx:16]: Here is new version: 
Screenshot 2024-10-02 at 14 48 32

[the remark this answers is not in the document]
Comment (atimin) [reply src/components/PricingTable/index.tsx:16]: Agree, let's keep your version.

File: src/pages/solutions/cloud/index.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+const cloudFaqs = [
+  {
+    question: "How does ReductStore optimize data management for the cloud?",
+    answer:
+      "ReductStore is designed to handle high-frequency time series data and unstructured blobs, ensuring fast access and processing in cloud environments. It's optimized for AI and IoT applications, providing scalable and efficient data storage.",
+  },
+  {
+    question: "What makes ReductStore secure in the cloud?",
+    answer:
+      "ReductStore Cloud is built on Google infrastructure with regional isolation, deep security controls, and SOC2 certification to ensure robust protection. Deploy on our account or yours and manage access with granular permissions.",
```
Comment (atimin) [inline src/pages/solutions/cloud/index.tsx:110]: it's not true.. we will spin up a cluster only in one region. And all our customers will be together =D.   i'd just remove it sofar.

File: src/pages/solutions/cloud/index.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      .default,
+    isImageLeft: true,
+  },
+];
+
+const cloudFaqs = [
+  {
+    question: "How does ReductStore optimize data management for the cloud?",
+    answer:
+      "ReductStore is designed to handle high-frequency time series data and unstructured blobs, ensuring fast access and processing in cloud environments. It's optimized for AI and IoT applications, providing scalable and efficient data storage.",
```
Comment (atimin) [inline src/pages/solutions/cloud/index.tsx:105]: Let's add that we use Google Cloud Storage to make the solution scalable and cost effective.

**Review by atimin — CHANGES_REQUESTED** [review]

**Comment by atimin** [comment]
image

The dark theme is hard to read.

**Review by atimin — APPROVED** [review]
Perfect!

**Pull request merged**